[OT294-74]-Endpoint para listar miembros

LuciaCerpa opened the pull request:
https://alkemy-labs.atlassian.net/jira/software/c/projects/OT294/boards/388?modal=detail&selectedIssue=OT294-74

![Endpoint_Listar-Miembros](https://user-images.githubusercontent.com/102775738/193116621-ff22a9fc-7f65-42e2-b3f7-e28229baf6f2.png)

lucy1528 committed:
Endpoint para listar miembros

FerLeiva commented on controllers/members.controller.js:
```diff
@@ -0,0 +1,17 @@
+const { catchAsync } = require("../utils/catchAsync.util");
+
+const getAllMembers = catchAsync(async (req, res) => {
+    const membersAll = await members.findAll();
```
allMembers es un mejor nombre

LuciaCerpa replied:
Cambios listos

LuciaCerpa replied:
![cambioDeNombreDeVariable](https://user-images.githubusercontent.com/102775738/193319675-2f6f2b03-741c-49f2-a7d3-df787e07ae34.png)

FerLeiva commented on controllers/members.controller.js:
```diff
@@ -0,0 +1,17 @@
+const getAllMembers = catchAsync(async (req, res) => {
+    const membersAll = await members.findAll();
+
+    res.status(201).json({
```
El status 201 CREATED es para, indica que la solicitud ha tenido éxito y ha llevado a la creación de un recurso. Como esto es un GET, lo correcto es que devuelva un status 200 OK

LuciaCerpa replied:
Cambios listos

LuciaCerpa replied:
![cambioRespuesta](https://user-images.githubusercontent.com/102775738/193319324-484a430d-b12f-464b-81ed-0eaa9d28909d.png)

FerLeiva reviewed (changes requested):
Ver comentarios

LuciaCerpa committed:
Update members.controller.js

cambio de nombre de la variable en respuesta y cambio de código de estado de la respuesta
